docs/mobile/exercises/README.md

```diff
@@ -170,36 +170,27 @@ image: assets/images/my-cool-plot.png
 ```
 
 You can insert an image block into your exercise simply by specifying the path
-to a PNG file. This is useful when teaching things like plotting. Be mindful of
-image asset size, because these images will be downloaded to students' phones,
-often over a cellular data connection.
+to a PNG file. The image will be placed below the question. This is useful when
+teaching things like plotting. Be mindful of image asset size, because these
+images will be downloaded to students' phones, often over a cellular data
+connection.
 
-## complex context
+![Example image](/images/mobile/image-small.png)
 
-The `context` block can be more complex than a simple string. Sometimes you need
-to introduce a table, image, multi-line code snippet, or output in the context,
-*before* the question block. This really starts to stretch the limits of what
-can be displayed on a single screen, so simple text context should always be
-strongly preferred.
-
-For example, to include an image such as a plot:
+## context image
 
 ```yaml
 context:
-  text: Study the plot below to answer the following question.
-  image: assets/images/my-cool-plot.png
+  text: "Below is a line plot of life expectancy for different countries against their GDP per capita."
+  image: assets/gdp-life-line.png
 ```
 
-To include a table:
+If you want your image to appear _above_ the question, you can use a
+**multi-part** context block rather than a simple text one. To do so, put the
+text of your context into a subfield called `text`, and then add nest `image`
+field into `context` as well, as demonstrated above.
 
-```yaml
-context:
-  text: Study the table below to answer the following question.
-  table: |-
-    foo, bar, baz
-    1  ,2   ,3
-    4  ,5   ,6
-```
+![Example context image](/images/mobile/image-context-small.png)
 
 ## feedback
```

Comment:
Don't mention this maybe? I thought we decided we would not longer support it?

Also, how it works right now is that feedback is ALWAYS shown, UNLESS overwritten by feedback_wrong or distractors.

Reply:
So if an instructor only provides feedback and not feedback_wrong or distractors, and the user answers incorrectly, he will see the message provided in feedback (which might be 'great, well done!' :p)

Reply:
Seems like that behavior would never be desired, but yes this is unsupported for now and I forgot to remove it.